org.musicbrainz.Picard.appdata.xml

```diff
@@ -1,20 +1,57 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<application>
-  <id type="desktop">picard.desktop</id>
+<component type="desktop-application">
+  <id>org.musicbrainz.Picard</id>
+  <project_license>GPL-2.0+</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
   <name>Picard</name>
   <summary>MusicBrainz's music tagger</summary>
+
   <description>
     <p>
       Tag your music with the next generation MusicBrainz tagger.
     </p>
   </description>
-  <url type="homepage">https://picard.musicbrainz.org/</url>
+
+  <launchable type="desktop-id">picard.desktop</launchable>
```

Comment:
That needs to be `org.musicbrainz.Picard.desktop`.

Reply:
`rename-desktop-file` *should* catch this but I didn't check the output personally.

Reply:
> rename-desktop-file should catch this but I didn't check the output personally.

I checked, it does not.

Should I open a bug in flatpak-builder?